apps/server/src/shared/repo/news/news-scope.spec.ts

```diff
@@ -82,7 +82,7 @@ describe('News Scope', () => {
 		});
 		it('should create correct query for unpublished', () => {
```

Comment:
here are missing some tests for byUnpublished and byCreator

Reply:
Added tests